README.md

```diff
@@ -41,6 +41,33 @@ Using method extraction to call an object method when a DOM event occurs:
 $(".some-link").on("click", ::view.reset);
 ```
 
+#### How it works
+
+The `::` operator has the highest precedence. And so, the following:
```

Comment:
"highest precedence" may be confusing.  It should be higher than function call operator `()`, and lower than property access operator `.`.